src/Conversion/ONNXToKrnl/Additional/Custom.cpp

```diff
@@ -20,6 +20,84 @@ using namespace mlir;
 
 namespace onnx_mlir {
 
+void FixGRUY(Location loc, ConversionPatternRewriter &rewriter,
```

Comment:
Is it still necessary? I think it's redundant.

Reply:
Yes, they are redundant.